fields/types/list/ListColumn.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ItemsTableCell from '../../components/ItemsTableCell';
+import ItemsTableValue from '../../components/ItemsTableValue';
+import { plural } from '../../../admin/client/utils/string';
+
+var ListColumn = React.createClass({
```

Comment:
Babel included in package.json, so normally ES6 syntax will be supported.

> export default class ListColumn extends React.Component {
> ...
> }

Remove module.exports